docs_src/async_sql_databases/tutorial001.py

```diff
@@ -58,8 +58,35 @@ async def read_notes():
     return await database.fetch_all(query)
 
 
+@app.get("/notes/{id}/", response_model=Note)
+async def read_one_note(id: int):
```

Comment:
use note_id instead of id to not trigger built-in id warning

Reply:
I have changed it to `note_id` in latest series of commits.